v2/googlecloud-to-splunk/docs/GCSToSplunk/README.md

```diff
@@ -0,0 +1,109 @@
+# GCS To Splunk Dataflow Template
+
+The [GCSToSplunk](../../src/main/java/com/google/cloud/teleport/v2/templates/GCSToSplunk.java) pipeline ingests
+data from CSV files stored in GCS, optionally executes a UDF,
+and writes those records into Splunk's HEC endpoint.
+
+The template creates the Splunk payload as a JSON element using one of the following:
+
+1. CSV headers* (default option, used only if none of the below two inputs were provided)
```

Comment:
What is the asterisk for?

I also think "default" is enough. There's no need to say that it will be used if the other inputs aren't provided, since that's implied by "default". You can also probably remove the "(if provided)" on the other two for the same reason.